elliottlib/bzutil.py

```diff
@@ -464,6 +483,41 @@ def filter_bugs_by_cutoff_event(self, bugs: Iterable, desired_statuses: Iterable
     def advisory_bug_ids(self, advisory_obj):
         return advisory_obj.jira_issues
 
+    def get_corresponding_flaw_bugs(self, tracker_bugs: List[JIRABug], flaw_bug_tracker: BugTracker = None,
+                                    strict: bool = False):
+        """Get corresponding flaw bug objects for given list of tracker bug objects.
+        Accepts a flaw_bug_tracker object to fetch flaw bugs from incase it's different from self
+
+        :return: (tracker_flaws, flaw_id_bugs): tracker_flaws is a dict with tracker bug id as key and list of flaw
+        bug id as value, flaw_id_bugs is a dict with flaw bug id as key and flaw bug object as value
+        """
+        bug_tracker = flaw_bug_tracker if flaw_bug_tracker else self
+        flaw_bugs = bug_tracker.get_bugs(list(set(sum([t.corresponding_flaw_bug_ids for t in tracker_bugs], []))))
```

Comment:
the only difference is jirabug.corresponding_flaw_bug_ids and buzillabug.blocks? can we change the define of jirabug.blocks to align with it here and make currently jirabug.blocks into another name?
then can we combine two separate functions into a common function

Reply:
That's a good point, let me see. I'd rather change bugzilla.blocks to corresponding_flaw_bug_ids since that explains what we want out of it

Reply:
Alright I've consolidated the get_corresponding_flaw_bugs(), and broken out the specific logic into 2 separate methods - get_tracker_bugs() and get_flaw_bugs(). Thanks for the suggestion, it looks much cleaner now!